haswell/lib_haswell_memory_bound.cpp

```diff
@@ -334,32 +524,34 @@ extern "C" {
     int allinea_plugin_cleanup(plugin_id_t plugin_id, void *unused)
     {
         // Stop the event set counting
-        int retval= PAPI_stop(gEventSet, gEventValues.data());
-        if (retval != PAPI_OK)
-        {
-            allinea_set_plugin_error_messagef(plugin_id, retval, "Error in PAPI_stop: %s", PAPI_strerror(retval));
-            return ERROR;
-        }
-        // Remove all events from the event set
-        retval = PAPI_cleanup_eventset(gEventSet);
-        if (retval != PAPI_OK)
-        {
-            allinea_set_plugin_error_messagef(plugin_id, retval, "Error in PAPI_cleanup_eventset: %s", PAPI_strerror(retval));
-            return ERROR;
-        }
-
-        // Destroy the event set
-        retval = PAPI_destroy_eventset(&gEventSet);
-        if (retval != PAPI_OK)
-        {
-            allinea_set_plugin_error_messagef(plugin_id, retval, "Error in PAPI_destroy_eventset: %s", PAPI_strerror(retval));
-            return ERROR;
-        }
-
-        // Reset the event set
-        gEventSet= PAPI_NULL;
-
-        return 0;
+      int retval;
+      if (ARM_MAP_MEMORY_BOUND)
+	retval= PAPI_stop(gEventSet, MB::gEventValues.data());
```

Comment:
Indentation looks messed up, but again, might just be github.

Reply:
Tabs again.